requirements-ci-oldest.txt

```diff
@@ -4,7 +4,6 @@
 #
 #    pip-compile --extra-index-url=https://download.pytorch.org/whl/cpu --extra=ann --extra=docs-additional --extra=dune --extra=gmsh --extra=gui --extra=ipyparallel --extra=jupyter --extra=mpi --extra=ngsolve --extra=pymess --extra=scikit-fem --extra=slycot --extra=tests --extra=vtk --output-file=requirements-ci-oldest.txt --resolver=backtracking pyproject.toml requirements-ci-oldest-pins.in
 #
-#
```

Comment:
This causes the old image with the too-old slycot to be picked up again. (You can also try manually removing the image from the registry to force a rebuild.)

Reply:
Ok, makes sense. I removed the image.